src/components/List/__tests__/index.test.tsx

```diff
@@ -24,15 +32,19 @@ describe('List', () => {
           <>
             <list.Header />
             <list.Body>
-              {({ rowData }) => (
-                <list.Row id={rowData.id}>
-                  <list.Cell>{rowData.name}</list.Cell>
-                  <list.Cell>{rowData.description}</list.Cell>
-                  <list.Cell>{rowData.department}</list.Cell>
-                  <list.Cell>{rowData.reference}</list.Cell>
-                  <list.Cell>actions</list.Cell>
-                </list.Row>
-              )}
+              {props => {
+                const { rowData } = props as ListBodyRenderProps<ListRowData>
```

Comment:
I tried to improve this (use type inference to avoid having typing `rowData`) but couldn't do better...